dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/sanitizer/NoSchemaSanitizerStrategy.kt

```diff
@@ -0,0 +1,27 @@
+package com.datadog.gradle.plugin.internal.sanitizer
+
+import java.net.URI
+
+internal class NoSchemaSanitizerStrategy : SanitizerStrategy {
+    override fun apply(url: String): String {
```

Comment:
This one could be simplified by doing 
```kotlin
internal class ShortUriUserInfoSanitizer : FullUriUserInfoSanitizer {
    override fun apply(url: String): String {
        return super.apply("$SSH_SCHEMA_PREFIX$url")
                    .removePrefix(SSH_SCHEMA_PREFIX)
    }
}
```

Reply:
In the same time this will only work with one specific prefix : `ssh:\\` in this case, I'd rather keep it like this which applies for all prefixes and it is also one operation in minus having not to do that `search and replace` in the String at the end.